.github/bin/s3/setup-empty-s3-bucket.sh

```diff
@@ -10,10 +10,18 @@ S3_BUCKET_IDENTIFIER=trino-s3fs-ci-$(openssl rand -hex 8)
 S3_BUCKET_TTL=$(date -u -d "+2 hours" +"%Y-%m-%dT%H:%M:%SZ" 2>/dev/null || date -u -v "+2H" +"%Y-%m-%dT%H:%M:%SZ")
 
 echo "Creating an empty AWS S3 bucket ${S3_BUCKET_IDENTIFIER} in the region ${AWS_REGION}"
+
+# LocationConstraint configuration property is not allowed for us-east-1 AWS region
+if [ "${AWS_REGION}" == 'us-east-1' ]; then
+    optBucketConfiguration=("")
```

Comment:
This should be an empty array, not an array with an empty string

Reply:
updated